components/__tests__/HomeScreen-test.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import HomeScreen from '../../screens/HomeScreen'
+
+import renderer from 'react-test-renderer';
+
+test('renders correctly', () => {
+  const tree = renderer.create(<HomeScreen />).toJSON();
+  expect(tree).toMatchSnapshot();
+  
+
+
+
+});
```

Comment:
Are these functioning and if not is there a reason they are being added?

Reply:
have removed and re-pushed.